cmd/podman/build.go

```diff
@@ -145,22 +142,41 @@ func buildCmd(c *cli.Context) error {
 	}
 	defer runtime.Shutdown(false)
 
+	var stdout, stderr, reporter *os.File
```

Comment:
Does this address the blank line in the logs error?

Reply:
I think that is the goal.  Although I want @baude to confirm.

Reply:
I just wanted podman build to match up with what buildah bud is doing.